rdkit/Chem/Draw/MolDrawing.py

```diff
@@ -416,27 +416,26 @@ def AddMol(self, mol, centerIt=True, molTrans=None, drawingTrans=None, highlight
           symbolLength = len(symbol)
         else:
           base = atom.GetSymbol()
-          if (base == 'H' and (iso == 2 or iso == 3) and
-              self.drawingOptions.atomLabelDeuteriumTritium):
+          if base == 'H' and (iso == 2 or iso == 3) and self.drawingOptions.atomLabelDeuteriumTritium:
             if iso == 2:
               base = 'D'
             else:
               base = 'T'
             iso = 0
           symbolLength = len(base)
+          
+          nHs = 0
+          hs = ''
           if not atom.HasQuery():
             nHs = atom.GetTotalNumHs()
-          else:
-            nHs = 0
           if nHs > 0:
             if nHs > 1:
               hs = 'H<sub>%d</sub>' % nHs
               symbolLength += 1 + len(str(nHs))
             else:
               hs = 'H'
               symbolLength += 1
-          else:
-            hs = ''
+
```

Comment:
You can convert the `if-if-else` block into a `if-elif` block and make the block clearer
```
          nHs = 0
          if not atom.HasQuery():
            nHs = atom.GetTotalNumHs()
          hs = ''
          if nHs > 1:
            hs = 'H<sub>%d</sub>' % nHs
            symbolLength += 1 + len(str(nHs))
          elif nHs == 1:
            hs = 'H'
            symbolLength += 1
```
I moved the hs assignment before the `if-elif` block as it belongs to it. First part deals with `nHs` the second part with `hs`.

Reply:
Agree ! I would fix it in the next patch